### ISSUE TITLE
Can't craft thrusters or jetpacks.

### ISSUE THREAD

**Issue by Vrokxx** [CyberdyneCC/SimplyJetpacks-2#119]
Hi, I am having a problem on my modpack I have made, not being able to craft thrusters or jetpacks. I am not sure wether its server side or client side but thrusters cant be crafted in a crafting table and if I spawn in the thrusters the jetpack cant be made. Im wondering if its an ID clash somewhere but not sure how to check.
Thanks

**Comment by Tomson124** [comment]
Could you please provide a mod list. And did you try to only use SJ2 and its depdencies like EnderIO or ThermalExpansion?

**Comment by Vrokxx** [comment]
Here is the modlist... I havent tried it by itself yet I will try now
https://pastebin.com/VHvvBYF3

**Comment by Vrokxx** [comment]
I tried with just SJ2 and its dependencies, (enderio, thermal expansion/dynamics/foundation, redstone flux) and I still cant seem to craft them? The recipes dont show up in the crafting guide and when I try to find a crafting recipe online they are all years outdated or they just dont work... so im not sure if its just me crafting them wrong or if its something in a config

**Comment by Tomson124** [comment]
Try to install JEI, too, as the crafting guide is not working correct all the times. And also try deleting the SJ2 configs before starting mincraft.

**Comment by Vrokxx** [comment]
Ok it works on the test server I setup with the dependencies and JEI but it doesnt work on my modpack server...must be an id clash right?

**Comment by Tomson124** [comment]
Yeah it probably is. What you could try if you have the time is start only with SJ2 and the dependencies and then add mod after mod to his setup until it doesn't work anymore, then we know what is causing it.
